Megaparsec/MonadParsec.lean

```diff
@@ -238,8 +242,12 @@ instance theInstance {m : Type u → Type v} {α β ℘ E : Type u}
     match y.1 with
     | .nil => eok.2 default {s with input := y.2} hs
     -- TODO: Maybe it's just <$> for Chunk? Do we have Functor? I forget.
-    | .cont cs => cok.2 cs  {s with input := y.2, offset := s.offset + Iterable.length cs} hs -- TODO: Why hs, not [] ?
-    | .fin (cs, _) => cok.2 cs  {s with input := y.2, offset := s.offset + Iterable.length cs} hs -- TODO: Why hs, not [] ?
+    | .cont cs =>
+      let offset' := s.offset + Iterable.length cs
```

Comment:
I don't understand this implementation btw, so if you or someone can explain it, it'd be nice to put a comment or docstring here.